common/configure

```diff
@@ -123,14 +133,8 @@ if [ -n "$unknown_args" ]; then
     echo "Unknown Arguments: $unknown_args"
 fi
 
-#patch python command
-#use 'python' or 'python3' to start Python Version 3.x
-case $PYTHON in
-    --python)   PYVERSION="" ;;
-    --python3)  PYVERSION="3";;
-esac
-sed -e "s/^python3\? /python${PYVERSION} /g" \
-    -e "s/^ssh-agent python3\? /ssh-agent python${PYVERSION} /g" \
+sed -e "s#^python3 #${PYTHON} #g" \
```

Comment:
Why is `#` instead of `\` used as substitution delimiter?

Reply:
Indeed, any character can be used as long (as it doesn't appear in `regexp`, of course) -- I picked `#` since that was already used in qt/configure for the same purpose.